docs/container_setup/github_codespace/codespacestartstop.md

```diff
@@ -25,7 +25,7 @@ The code changes and operations we have performed inside the codespace will stil
 
 - Go to [github.com/codespaces](https://github.com/codespaces)
 - Here we can see a list of the codespaces we have created
-  ![stop codespace](../assets/rdev17.png)
+  ![stop codespace](../../assets/rdev17.png)
 - To restart it, we can just click on the codespaces we wanted to use and it will start the codespaces again for us.
 - You can also see an active label added to the codespaces we just started
- ![stop codespace](../assets/rdev18.png)
+ ![stop codespace](../../assets/rdev18.png)
```

Comment:
Here you can add a similar section on workspace management to the one you should add to the Gitpod start/stop page, addressing my earlier review comment:

> GitHub Codespaces has a similar issue [regarding auto-deletion]: https://docs.github.com/en/codespaces/setting-your-user-preferences/configuring-automatic-deletion-of-your-codespaces#avoiding-automatic-deletion-of-codespaces